test-cargo-miri/Cargo.toml

```diff
@@ -16,6 +16,7 @@ issue_1567 = { path = "issue-1567" }
 issue_1691 = { path = "issue-1691" }
 issue_1705 = { path = "issue-1705" }
 issue_1760 = { path = "issue-1760" }
+issue_2705 = { path = "issue-2705" }
```

Comment:
I was expecting we would add something somewhere that calls `cargo miri run -Zbuild-std`; why is this a separate crate?